src/components/RelatedAgreementsFieldArray/RelatedAgreementField.js

```diff
@@ -144,53 +158,63 @@ export default class RelatedAgreementField extends React.Component {
         </Row>
       </div>
     );
-  }
+  };
 
-  renderEmpty = () => (
-    <div>
-      <Layout className="textCentered">
-        <strong>
-          <FormattedMessage id="ui-agreements.relatedAgreements.noneLinked" />
-        </strong>
-      </Layout>
-      <Layout className="textCentered">
-        <FormattedMessage id="ui-agreements.relatedAgreements.linkToStart" />
-      </Layout>
-    </div>
-  )
+  const renderEmpty = () => {
+    return (
```

Comment:
With functional components you can exchange

```
const functionName = () => {
  return (
    "hello  world"
  );
}
```
with

```
const functionName = () => (
  "hello world"
);
```

Ie if your function only consists of a return block, you can return straight away.

Reply:
I think you can change the above function `renderAgreement` as well